sdk/communication/Azure.Communication.NetworkTraversal/tests/CommunicationRelayClient/CommunicationRelayClientLiveTests.cs

```diff
@@ -124,6 +124,12 @@ public async Task GettingTurnCredentialsGeneratesTurnCredentialsWithoutIdentity(
             Assert.IsNotNull(turnCredentialsResponse.Value);
             Assert.IsNotNull(turnCredentialsResponse.Value.ExpiresOn);
             Assert.IsNotNull(turnCredentialsResponse.Value.IceServers);
+
+            var configJ = turnCredentialsResponse.Value;
+            var jsonResponse = JsonSerializer.Serialize(configJ);
+            var relayConfig = JsonSerializer.Deserialize<CommunicationRelayConfiguration>(jsonResponse);
```

Comment:
Is it worth making this a separate test and doing a deep comparison to ensure serialization round trips?  Hopefully that'd help catch any problems if something failed to serialize/deserialize correctly in future API additions.